apps/hash-integration-worker/src/util.ts

```diff
@@ -0,0 +1,196 @@
+import { VersionedUrl } from "@blockprotocol/type-system";
```

Comment:
This file and `mappings.ts` are private dependencies of `activities.ts` – can we move them into a subfolder? Suggest also renaming this to something more specific (trying to avoid generic 'util' dumping grounds), e.g.
```
activities.ts
activities/
  mappings.ts
  graph-requests.ts
```

At some point we'll have to figure out organizing this worker for multiple integrations. maybe something like this (not necessary now)
```
activities.ts
activities/
  linear.ts
    linear/
      mappings.ts
  github.ts
     github/
       mappings.ts
```

Reply:
Addressed in https://github.com/hashintel/hash/pull/3636/commits/8d51ecc02aa2f662f4b9cfb634c5771245a20256